analytical/providers/googleanalytics.py

```diff
@@ -29,38 +28,39 @@ class Provider(BaseProvider):
     DEFAULT_DATA = {"v": "1"}  # analytics version (always 1)
 
     def __init__(
-        self, initial_data, timeout=DEFAULT_TIMEOUT, fail_silently=True, **kwargs
+        self,
+        initial_data,
+        timeout=DEFAULT_TIMEOUT,
+        fail_silently=True,
+        asynchronously=False,
```

Comment:
Wonder if this should just be `async` for spelling simplicity :)

Reply:
`async` is a reserved word in 3.7.

Reply:
That doesn't stop us from using it as a keyword argument to a function though, right? Just from using it as a variable name?

Reply:
Hrm, seems I'm wrong. Interesting.